### DISCUSSION TITLE
Restart script when internet connection is restored.

### DISCUSSION THREAD

**Discussion by dmytrokoren** [jdholtz/auto-southwest-check-in#275] [Ideas]
There should be a checker that pings every 10 minutes to see if the internet is disconnected and if the browser is unable to load the SW website. Currently, I have a script that runs every 4 hours, but if the internet connection is lost during this time, the script hangs while trying to load the SW website to get headers. Even after the internet connection is restored, the script remains stuck. Thus, the 4-hourly check doesn't run as it is still hanging from the previous attempt.

**Comment by jdholtz** [comment]
Where does it get stuck? When I disconnect it while it's loading the Southwest page, it waits forever for either the headers to be set or the login to go through. Do you see this as well? If so, this should be mitigated when I add a timeout as detailed in #249 (which should be soon after I merge #274). Then, the session would just timeout after about 5 minutes and move onto the next account/reservation or wait again for another 4 hours.

**Reply by jdholtz** [reply]
This should be fixed with the addition of #281. Let me know if you are still having issues in v8.0
